File: core/model/flow_info.go
package model

type FlowInfo struct {
	BaseModel
	FlowKey  string `gorm:"column:flow_key;size:20" json:"flowKey"`
	FlowName string `gorm:"column:flow_name;size:60" json:"flowName"`
	FlowType string `gorm:"column:flow_type;size:8" json:"flowType"`
	Remark   string `gorm:"column:remark;size:200" json:"remark,omitempty"`
}

func (FlowInfo) TableName() string { return "t_flow_info" }

File: commom/src/main/java/com/javaoffers/nexus/common/model/FlowInfo.java
package com.javaoffers.nexus.common.model;

import com.javaoffers.brief.modelhelper.anno.BaseModel;
import com.javaoffers.brief.modelhelper.anno.BaseUnique;
import com.javaoffers.brief.modelhelper.anno.ColName;
import com.javaoffers.brief.modelhelper.anno.derive.flag.IsDel;
import lombok.Data;

import java.util.Date;

@BaseModel("t_flow_info")
@Data
public class FlowInfo {
    @BaseUnique
    private Long id;
    private IsDel deleted;
    private Date createdAt;
    private Long createdBy;
    private Date updatedAt;
    private Long updatedBy;

    @ColName(value = "id", excludeColAll = true)
    private Long countId;

    private String flowKey;
    private String flowName;
    private String flowType;
    private String remark;
}
